### DISCUSSION TITLE
关于运行时缺少模块导致崩溃的问题

### DISCUSSION THREAD

**Discussion by iamlivehaha** [BoomingTech/Piccolo#263] [Q&A] [answered]
**Describe the bug | Bug描述**
工程项目在成功build后开始运行，成功运行后大概10s中崩溃并报错

**Steps to reproduce | 如何复现**
1. 下载git release 工程文件
2. 运行build_windows.bat
3. 打开vs工程项目
4. debug运行

**Computer Configuration | 电脑配置**
![image](https://user-images.githubusercontent.com/30818771/172978558-238bf0e0-28e8-4e50-b66e-7d92db889312.png)

**Screenshots/Logs | 截图/日志**
![image](https://user-images.githubusercontent.com/30818771/172978148-fd184b80-567b-4c99-93cd-e2efa1cfc46f.png)
![image](https://user-images.githubusercontent.com/30818771/172978196-ecee0786-c840-4e4b-a651-baf1d3f620d4.png)
![image](https://user-images.githubusercontent.com/30818771/172978250-ff6fd642-1b39-4a73-a8d8-cd3a389c8746.png)
![image](https://user-images.githubusercontent.com/30818771/172978307-f353e88c-b16f-4483-abd1-e77754007f3b.png)

**Comment by BoomingTechDev** [comment]
可以看到callstack吗

**Reply by iamlivehaha** [reply]
是这个嘛？
![image](https://user-images.githubusercontent.com/30818771/173063578-2404da8c-d9a9-470a-a9d8-b232e0b1681e.png)

**Reply by ShenMian** [reply]
是的, 这是在 Debug 配置下编译的吗?

**Reply by iamlivehaha** [reply]
是的

**Reply by hyv1001** [reply]
Callstack比较奇怪，理论上不应该找不到引擎中代码的符号。
但看样子是崩在NV驱动中了，可以尝试：
1. 更新下N卡驱动；
2. 下载安装最新的[VulkanSDK](https://vulkan.lunarg.com/)

**Reply by iamlivehaha** [reply]
N卡驱动到了最新
vulkanSDK也是1.3.211.0版本
但还是崩溃。。。

**Comment by iamlivehaha** [comment, marked as answer]
Bug Solved！
Inspired by similar problem 
**[[Crash in nvogl64.dll creating a vulkan ray-tracing pipeline with raygen shader using ray queries]](https://forums.developer.nvidia.com/t/crash-in-nvogl64-dll-creating-a-vulkan-ray-tracing-pipeline-with-raygen-shader-using-ray-queries/198945)**

在debug模式下跟踪模块加载窗口，发现了奇怪的东西加载进入引擎
![image](https://user-images.githubusercontent.com/30818771/173490696-0beb427a-e444-416c-b8d2-062beb6a304f.png)

结果是这个软件会跟踪游戏加载错误的shader模块？
![image](https://user-images.githubusercontent.com/30818771/173490748-ccbefd45-6007-47ce-b7ca-025038f5a933.png)

**Bug Solution | 解决方案**
关闭 “游戏加加” 软件
